buildlib/azure-pipelines-pr.yml

```diff
@@ -134,6 +134,7 @@ stages:
 
           - bash: |
               set -eE
+              set -o pipefail
```

Comment:
set -o pipefail before "make -j`nproc` 2>&1 | tee compile.log"
and set +o pipefail right after it
maybe it affects cslinker ... command